src/Workspaces/CoreTest/UtilityTest/FilePathUtilitiesTests.cs

```diff
@@ -99,5 +99,27 @@ public void GetRelativePath_WithBaseDirectoryMatchingIncompletePortionOfFullPath
 
             Assert.Equal(expected: @"..\Beta2\Gamma", actual: result);
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72043")]
+        public void GetRelativePath_CrashingWithTwoUp()
```

Comment:
Need to test the Linux version of this problem in addition to Windows. Recommend making this a `[Theory]` and then use windows and linux paths as the inputs.